src/helix.vcl

```diff
@@ -0,0 +1,1216 @@
+                    opacity: 0.6; 
+                    -webkit-animation: moveclouds 25s linear infinite;
+                    -moz-animation: moveclouds 25s linear infinite;
+                    -o-animation: moveclouds 25s linear infinite;
+                }
+
+                .x3 {
+                    left: 250px; bottom: -70px;
+
+                    -webkit-transform: scale(0.6);
+                    -moz-transform: scale(0.6);
+                    transform: scale(0.6);
+                    opacity: 0.8; 
+
+                    -webkit-animation: moveclouds 25s linear infinite;
+                    -moz-animation: moveclouds 25s linear infinite;
+                    -o-animation: moveclouds 25s linear infinite;
+                }
+
+                .x4 {
+                    left: 470px; botttom: 20px;
+
+                    -webkit-transform: scale(0.75);
+                    -moz-transform: scale(0.75);
+                    transform: scale(0.75);
+                    opacity: 0.75;
+
+                    -webkit-animation: moveclouds 18s linear infinite;
+                    -moz-animation: moveclouds 18s linear infinite;
+                    -o-animation: moveclouds 18s linear infinite;
+                }
+
+                .x5 {
+                    left: 200px; top: 300px;
+
+                    -webkit-transform: scale(0.5);
+                    -moz-transform: scale(0.5);
+                    transform: scale(0.5);
+                    opacity: 0.8; 
+
+                    -webkit-animation: moveclouds 20s linear infinite;
+                    -moz-animation: moveclouds 20s linear infinite;
+                    -o-animation: moveclouds 20s linear infinite;
+                }
+
+                @-webkit-keyframes moveclouds {
+                    0% {margin-left: 1000px;}
+                    100% {margin-left: -1000px;}
+                }
+                @-moz-keyframes moveclouds {
+                    0% {margin-left: 1000px;}
+                    100% {margin-left: -1000px;}
+                }
+                @-o-keyframes moveclouds {
+                    0% {margin-left: 1000px;}
+                    100% {margin-left: -1000px;}
+                }    
+        </style>
+      </head>
+      <body>
+        <div id="clouds">
+          <div class="cloud x1"></div>
+          <div class="cloud x1_5"></div>
+          <div class="cloud x2"></div>
+          <div class="cloud x3"></div>
+          <div class="cloud x4"></div>
+          <div class="cloud x5"></div>
+      </div>
+      <div class='c'>
+          <div class='_404'>500</div>
+          <hr>
+          <div class='_1'>INTERNAL</div>
+          <div class='_1'>SERVER ERROR</div>
+          <code class='btn'>There is no action with the configured prefix.</code>
+      </div>
+      </body>
+    </html>"};
+    return(deliver);
+  }
+  if (obj.status == 953 ) {
+    set obj.http.Content-Type = "text/html";
+    synthetic {"<!doctype html>
```

Comment:
is there a better way to do this? maybe use some sort of `include` ?
having this large vlc is rather hard to understand and maintain. also, inline HTML is rather nasty.

Reply:
I think I can generate the `synthetic` block from example html files and then include it. Only downside is that we will have to make additional requests to create these VCL files.